README.md

```diff
@@ -38,6 +40,7 @@ Operational knowledge of:
   - #### Docker
     - [Docker](https://docs.docker.com/get-docker/)
 
+Note, this project has been tested and verified with these versions.
```

Comment:
Not sure the real reason for this addition? The version are listed above and if they are not used things might break. This line can be removed.

Reply:
The reason for the line was because I had to ask you and Norm if the versions were greater than or equal and you both responded that the required versions are equal.  So I added the line to ensure the users understand it was a hard requirement of the version numbers that were validated.

Reply:
The fact they are listed should be good enough. Please remove this is not helpful.